src/lib/inputs/Input.svelte

```diff
@@ -103,7 +111,15 @@
     {disabled}
     {...$$restProps}
     style={inputStyle}
+    readonly={readOnly}
     on:input={handleInput}
+    on:input
+    on:focus
+    on:blur
+    on:change
```

Comment:
do we need on:input and on:change both?

Reply:
These are only places here as a "just in case". _If_ a user wants to use either/or it is now available.